CakePHP/tests/files/function_comment_opening_line_fail.php

```diff
@@ -1,13 +1,14 @@
 <?php
+namespace Beakman;
 
-class Foo {
+class Foo
+{
     /** 
-     * There is a (disallowed) trailing space after the opening ** in this doc block.
+     * ^ There is a (disallowed) trailing space after the opening ** in this doc block.
```

Comment:
To help point out the (disallowed) trailing space above it. :stuck_out_tongue_winking_eye: